plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -437,6 +437,7 @@ public PreparedStatement buildSql(ConnectorSession session, Connection connectio
             throws SQLException
     {
         PreparedQuery preparedQuery = prepareQuery(session, connection, table, Optional.empty(), columns, ImmutableMap.of(), Optional.of(split));
+        log.info(preparedQuery.getQuery());
```

Comment:
Remove this log line.

Reply:
Hi @sumannewton thanks for the review. Currently this is a draft since there is a major issue as I mentioned in #8404 , when it resolved I will clean up this code.